### PR TITLE
Created and Updated config.yml

### PR DESCRIPTION
**Pull request by shashanksjSSJ**
Hello @leecalcote, I have added config.yml from meshery/meshery, as the images were already updated I haven't made a lot of changes in them. I have submitted this for your review, and I want to thank @leecalcote for letting me contribute to the organization, this helped me to learn a lot.
Please inform me if there are any changes that I need to make. 
Signed-off-by: shashanksjSSJ <shashanksj26@gmail.com>

### CODE REVIEW

**Comment by leecalcote** [comment]
@ShivangShandilya ready for review.

**Comment by Ghat0tkach** [comment]
have you added meshery-celebration-dark and meshery-celebration-light svgs in .github/welcome/
?

**Comment by shashanksjSSJ** [comment]
@Ghat0tkach, when the background is light, shouldn't the logo be dark so that it could be visible to the users and same goes for the dark theme, it is light because the background-color is dark, if anything is wrong please inform me, the other change I make correct the changes now, sorry for the trouble, I am a newbie 😄

**Comment by shashanksjSSJ** [comment]
@Ghat0tkach, I have updated it like you told me to do, like I have described in my previous commit, I have copied and pasted the code from meshery/meshery therefore, I did not make any significant changes in the code as the images were working as expected, 😄 👍

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Thanks for your contribution to Meshery! :tada:
+   
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset=".github/welcome/Meshery_Welcome_light.png">
+  <source media="(prefers-color-scheme: light)" srcset=".github/welcome/Meshery_Welcome_light.png">
+  <img alt="Meshery Logo" src="">
+  </picture>
+
```
Comment (ShivangShandilya) [inline .github/config.yml:25]: Here I think @shashanksjSSJ , `srcset` needs to be a reference to absolute path to raw image.
Also the in the `srcset` both the images you have named as Meshery_Welcome_light so correct that too one should be named dark.
I'm also mentioning a PR where you can get better idea of what i'm talking about - [Meshkit - #308](https://github.com/meshery/meshkit/pull/308)

**Comment by shashanksjSSJ** [comment]
Hey @ShivangShandilya, I have uploaded the images into .github/welcome and then used raw image path for srcset, may be I need to change the image link /shashanksjSSJ/homebrew-tap/master/ to /mershery/homebrew-tap/master/, what do you think? 😅

File: .github/config.yml (hunk 1 above)
Comment (goldfishdolphin) [reply .github/config.yml]: @shashanksjSSJ for your contribution. 
Can you please use SVG format for the images and keep the files name in lowercase besides having an absolute path?

Comment (shashanksjSSJ) [reply .github/config.yml]: I need to change the raw link to /meshery/meshery right

**Comment by goldfishdolphin** [comment]
```

 <picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/homebrew-tap/master/.github/assets/images/meshery-celebration-dark.svg">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/home brew-tap/master/.github/assets/images/meshery-celebration-light.svg">
  <img alt="Meshery Logo" src="">
  </picture>
```

**Comment by shashanksjSSJ** [comment]
> 

@goldfishdolphin, I have used **https://raw.githubusercontent.com/meshery/homebrewtap/master/.github/assets/images/meshery-celebration-dark.svg** instead of
**https://raw.githubusercontent.com/shashanksjSSJ/homebrew-tap/Shashank-S-J-/.github/assets/images/meshery-celebration-dark.svg **
because after the merge happens the raw path also changes.

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+firstPRMergeComment: >
+  Thanks for your contribution to Meshery! :tada:
+   
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/homebrew-tap/master/.github/assets/images/meshery-celebration-dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/homebrew-tap/master/.github/assets/images/meshery-celebration-light.svg">
+  <img alt="Meshery Logo" src="">
+  </picture>
+
+  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; [Join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers). :smile:
```
Comment (goldfishdolphin) [inline .github/config.yml:26]: @thisiskaransgit  I am not sure if this line of code is needed here.

Comment (leecalcote) [reply .github/config.yml:26]: I wonder why we removed this from the other repos. It's helpful to both invite people to the Slack and to star the repo. I encourage us to leave this message included.

**Review by goldfishdolphin — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
Thank you, @goldfishdolphin

**Comment by leecalcote** [comment]
Good work, @shashanksjSSJ

**Pull request merged**